packages/fx-core/src/common/local/projectSettingLoader.ts

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+"use strict";
+import { AzureSolutionSettings, Context, v2 } from "@microsoft/teamsfx-api";
+import {
+  AadPlugin,
+  FunctionPlugin,
+  SpfxPlugin,
+  FrontendHostingPlugin,
+  BotPlugin,
+  RuntimeConnectorPlugin,
+} from "../../../plugins/resource/localdebug/constants";
+
+export class ProjectSettingLoader {
```

Comment:
Call it `ContextHelper`? Since the input is always ctx, no to load project settings file.

Reply:
Move this file to `packages/fx-core/src/plugins/solution/fx-solution/debug/`. Keep each PR as simple as possible.

Reply:
Got it, moved it to packages/fx-core/src/plugins/solution/fx-solution/debug/util/contextHelper.ts